deploy/docs/Security_Best_Practices.md

```diff
@@ -55,6 +157,47 @@ be able to reach the collector over the network.
 The tracing collector and metadata enrichment are currently done by the same service. As such, this service needs to
 be able to talk to the Sumo Logic backend receiver endpoints, and the Kubernetes API Server.
 
+Example Kubernetes Network Policies restricting all but the necessary traffic for the traces pipeline:
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: otelcol
+spec:
+  podSelector:
+    matchLabels:
+      app: <release_name>-sumologic-otelcol
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress:
+  - ports:
+    - protocol: TCP
+      port: 4317
+    - protocol: TCP
+      port: 5778
+    - protocol: TCP
+      port: 6831
+    - protocol: TCP
+      port: 6832
+    - protocol: TCP
+      port: 9411
+    - protocol: TCP
+      port: 14268
+    - protocol: TCP
+      port: 55678
+    - protocol: TCP
+      port: 55680
+    - protocol: TCP
+      port: 55681
+  egress: 
+  - {}
```

Comment:
What's this for?

Reply:
It means "allow all outbound connections". It's basically here for completeness' sake.